openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -2122,7 +2122,7 @@ boost::optional<IdfObject> ForwardTranslator::translateAndMapModelObject(ModelOb
       model::ShadingControl shadingControl = modelObject.cast<ShadingControl>();
       retVal = translateShadingControl(shadingControl);
       break;
-    }    
```

Comment:
@shorowit no i did not but that is awesome!

https://github.com/NREL/OpenStudio/pull/2254/files?w=1